examples/sort.py

```diff
@@ -87,7 +102,6 @@ def run_sort(
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "-c",
```

Comment:
Was there an issue with the single-character `-c` flag?

Reply:
The legate/python command is:
sort.py -s 20 -d double --gpus 1 -t --check **-cunumeric:test** --eager-alloc-percentage 1 -lg:sched 1024 -cunumeric:preload-cudalibs --eager-alloc-percentage 1 --fbmem 10288 -b 1
-cunumeric:test is for legate, but if we define -c as argument for python, it would report error:
argument -c/--check: ignored explicit argument 'unumeric:test'.
So we have to remove -c in argument. In this way, the error goes away.

Reply:
You are right. FYI, this is happening because now you are parsing the arguments before loading the cunumeric package, so cunumeric has not had a chance to remove its arguments from the command line.

Reply:
You are right. Thanks.